app/forms/nomination_form.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class NominationForm
+  include ActiveModel::Model
+  include ActiveModel::Serialization
+
+  attr_accessor :local_authority_id, :school_id
+  validates :local_authority_id,
+            presence: { message: "We could not find any local authorities matching your search criteria" },
+            on: %i[local_authority save]
+  validates :school_id,
+            presence: { message: "We could not find any establishments matching your search criteria" },
+            on: %i[school save]
+
+  def attributes
+    { local_authority_id: nil, school_id: nil }
+  end
+
+  def available_schools
+    SchoolLocalAuthority.where(local_authority_id: local_authority_id).joins(:school).includes(:school).map(&:school)
```

Comment:
I think this join should be the other way round i.e. school joins school local authority. That will give you what you need without having to use the includes or the map.